src/main/scala/uk/gov/nationalarchives/consignmentexport/Validator.scala

```diff
@@ -0,0 +1,53 @@
+package uk.gov.nationalarchives.consignmentexport
+
+import java.time.LocalDateTime
+import java.util.UUID
+
+import cats.effect.IO
+import graphql.codegen.GetConsignmentExport.getConsignmentForExport.GetConsignment.Files
+import cats.implicits._
+import graphql.codegen.GetConsignmentExport.getConsignmentForExport.GetConsignment
+import graphql.codegen.GetConsignmentExport.getConsignmentForExport.GetConsignment.Files.Metadata
+import uk.gov.nationalarchives.consignmentexport.Validator.ValidatedFileMetadata
+
+class Validator(consignmentId: UUID) {
+  def validateConsignmentHasFiles(consignmentData: GetConsignment): IO[Files] = {
+    IO.fromOption(consignmentData.files.headOption)(new Exception(s"Consignment API returned no files for consignment $consignmentId"))
+  }
+
+  def validateConsignmentResult(consignmentResult: Option[GetConsignment]): IO[GetConsignment] = {
+    IO.fromOption(consignmentResult)(new RuntimeException(s"No consignment metadata found for consignment $consignmentId"))
```

Comment:
Any particular reason for using `RuntimeException` here and `Exception` in `validateConsignmentHasFiles`?

Reply:
Consistency has never been one of my strengths :smile: I've moved them to all be `RuntimeException`